### DISCUSSION TITLE
Are wildcards supported for read-write folder?

### DISCUSSION THREAD

**Discussion by osevan** [netblue30/firejail#5085] [Q&A] [answered]
Is this allowed in syntax?

```
read-write /use/local/src/*
```

read-write for subfolders too?

Or how I can allow read and write subfolders?

**Comment by kmk3** [comment, marked as answer]
@osevan [on Mar 31](https://github.com/netblue30/firejail/discussions/5085#discussion-3977304):

> Is this allowed in syntax?
> 
> ```
> read-write /use/local/src/*
> ```

Yes; see firejail(1).

> read-write for subfolders too?
>
> Or how I can allow read and write subfolders?

Does your user have write permissions for /usr/local/src?

That path is usually not writable by unprivileged users and firejail is not
intended to bypass that.

I'd suggest trying `read-write` first with a path that is normally
user-writable, such as one in `${HOME}`.

**Reply by osevan** [reply]
> @osevan [on Mar 31](https://github.com/netblue30/firejail/discussions/5085#discussion-3977304):
> 
> > Is this allowed in syntax?
> > ```
> > read-write /use/local/src/*
> > ```
> 
> Yes; see firejail(1).
> 
> > read-write for subfolders too?
> > Or how I can allow read and write subfolders?
> 
> Does your user have write permissions for /usr/local/src?
> 
> That path is usually not writable by unprivileged users and firejail is not intended to bypass that.
> 
> I'd suggest trying `read-write` first with a path that is normally user-writable, such as one in `${HOME}`.

Yes its its writable for my user and works. 

Thank you very much

**Comment by osevan** [comment]
noblacklist too?

**Comment by osevan** [comment]
For noblacklist too?
